tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -102,41 +102,68 @@ Select "Pupil Exclusions" publication
     user clicks element   id:publicationSubjectForm-submit
     user waits until h2 is visible  Choose locations
     user checks previous table tool step contains  2    Subject     ${SUBJECT_NAME}
 
-Select Location Country, England
+Select Local Authority
+    [Tags]  HappyPath
+    user opens details dropdown     Local Authority
+    user clicks select all for category  Local Authority
+
+Select National
     [Tags]  HappyPath
-    user opens details dropdown     National
-    user clicks checkbox    England
+    user opens details dropdown  National
+    user clicks checkbox  England
+
+Select Regional
+    [Tags]  HappyPath
+    user opens details dropdown  Regional
+    user clicks select all for category  Regional
+
+Click next step button
     user clicks element     id:locationFiltersForm-submit
     user waits until h2 is visible  Choose time period
-    user checks previous table tool step contains  3   National     England
 
 Select Start date and End date
     [Tags]  HappyPath
-    user selects from list by label  id:timePeriodForm-start   2020 Week 14
-    user selects from list by label  id:timePeriodForm-end     2020 Week 14
+    user selects from list by label  id:timePeriodForm-start   2017/18
+    user selects from list by label  id:timePeriodForm-end     2017/18
     user clicks element     id:timePeriodForm-submit
     user waits until h2 is visible  Choose your filters
     user waits until page contains element   id:filtersForm-indicators
-    user checks previous table tool step contains  4    Start date    2020 Week 14
-    user checks previous table tool step contains  4    End date      2020 Week 14
+    user checks previous table tool step contains  4    Start date    2017/18
+    user checks previous table tool step contains  4    End date      2017/18
 
 Select Indicators
     [Tags]  HappyPath
-    user clicks indicator checkbox   Number of open settings
-    user checks indicator checkbox is checked  Number of open settings
-
-Select Filters
+    user clicks indicator checkbox   Authorised absence rate
+    user checks indicator checkbox is checked  Authorised absence rate
+    user clicks indicator checkbox   Number of persistent absentees
+    user checks indicator checkbox is checked  Number of persistent absentees
+    user clicks indicator checkbox   Number of pupil enrolments
+    user checks indicator checkbox is checked  Number of pupil enrolments
+    user clicks indicator checkbox   Number of schools
+    user checks indicator checkbox is checked  Number of schools
+    user clicks indicator checkbox   Overall absence rate
+    user checks indicator checkbox is checked  Overall absence rate
+    user clicks indicator checkbox   Percentage of persistent absentees
+    user checks indicator checkbox is checked  Percentage of persistent absentees
+    user clicks indicator checkbox   Unauthorised absence rate
+    user checks indicator checkbox is checked  Unauthorised absence rate
+
+Select gender filter
```

Comment:
I'd add more filters to this - although not a crazy amount. To test a subject with multiple filters thoroughly, we want to select a few and then look at the data in the resulting table and check it is correct.